### PR TITLE
Add .gitlab-ci.yml

### PR DESCRIPTION
**Pull request by jpetto**

### CODE REVIEW

**Comment by jgmize** [comment]
This depends on the deployment files listed in https://github.com/mozmeao/infra/issues/986#issuecomment-497047013

**Comment by jgmize** [comment]
As described in https://github.com/mozmeao/infra/issues/986#issuecomment-497399815 changing the the type selector in the deployments should unblock merging this.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  only:
+    - gitlab
+    - master
+  variables:
+    NAMESPACE: snippets-dev
+  script:
+    - bin/build_images.sh
+    - bin/push2dockerhub.sh
+    - CLUSTER_NAME=oregon-b bin/update-config.sh
+    - CLUSTER_NAME=frankfurt bin/update-config.sh
```
Comment (jgmize) [inline .gitlab-ci.yml]: There isn't a snippets-dev in frankfurt

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - meao
+    - aws
+  only:
+    - stage
+  variables:
+    NAMESPACE: snippets-stage
+  script:
+    - bin/build_images.sh
+    - bin/push2dockerhub.sh
+    - CLUSTER_NAME=frankfurt bin/update-config.sh
```
Comment (jgmize) [inline .gitlab-ci.yml]: There isn't a snippets-stage in frankfurt

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - aws
+  only:
+    - admin
+    - prod
+  variables:
+    NAMESPACE: snippets-admin
+  script:
+    - bin/build_images.sh
+    - bin/push2dockerhub.sh
+    - CLUSTER_NAME=frankfurt bin/update-config.sh
```
Comment (jgmize) [inline .gitlab-ci.yml]: there isn't a snippets-admin in frankfurt

**Comment by jpetto** [comment]
@jgmize Commented out config updates for missing namespaces.

**Pull request merged**